packages/expo-store-review/src/StoreReview.ts

```diff
@@ -1,9 +1,18 @@
 import Constants from 'expo-constants';
-import * as Linking from 'expo-linking';
 import { Platform } from 'expo-modules-core';
 
 import StoreReview from './ExpoStoreReview';
 
+type LinkingType = typeof import('expo-linking');
+let Linking: LinkingType | undefined;
+try {
+  Linking = require('expo-linking');
+} catch {
+  throw new Error(
```

Comment:
I don't think optional dependencies really work in Metro.

Reply:
we should just use React Native linking here imo, `expo-linking` isn't really adding anything afaict

Reply:
Sure, fixed